doc/rbd/nvmeof-target-configure.rst

```diff
@@ -0,0 +1,115 @@
+================================================
+Installing and Configuration the NVMe-oF Targets
+================================================
+
+Traditionally, block-level access to a Ceph storage cluster has been limited to
+QEMU and ``librbd``, which is a key enabler for adoption within OpenStack
+environments. Starting with the Ceph Reef release, block-level access has been 
+expanded to offer standard NVMe/TCP support, allowing wider platform usage and
+potentially opening new use cases.
+
+Prerequisites
+=============
+
+-  Red Hat Enterprise Linux/CentOS 8.0 (or newer); Linux kernel v4.16 (or newer)
+
+-  A working Ceph Storage cluster, deployed with ``cephadm``
+
+-  NVMe-oF gateways nodes, which can either be colocated with OSD nodes or on dedicated nodes
+
+-  Separate network subnets for NVME-oF front-end traffic and Ceph back-end traffic
+
+Explanation
+===========
+
+The Ceph NVMe/-oF gateway is both an NVMe-oF target and a Ceph client. Think of
+it as a "translator" between Ceph's RBD interface and the NVME-oF protocol. The
+Ceph NVMe-oF gateway can run on a standalone node or be colocated with other
+daemons, for example on a Ceph Object Store Disk (OSD) node. When colocating
+the Ceph NVMe-oF gateway with other daemons, ensure that sufficient CPU and
+memory are available. The steps below explain how to install and configure the
+Ceph NVMe/TCP gateway for basic operation.
+
+Requirements
+============
+
+-  A running Ceph Reef or later storage cluster
+
+-  Red Hat Enterprise Linux/CentOS 8.2 (or newer)
+
+Installation
+============
+
+Complete the following steps to install the Ceph NVME-oF gateway:
+
+#. Create a pool in which the gateways configuration can be managed:
+
+   .. prompt:: bash #
+
+      ceph osd pool create NVME-OF_POOL_NAME
+
+#. Enable RBD on the NVMe-oF pool:
+
+   .. prompt:: bash #
+   
+      rbd pool init NVME-OF_POOL_NAME
+
+#. Deploy the NVMe-oF manager daemons on a specific set of nodes:
+
+   .. prompt:: bash #
+   
+      ceph orch apply nvmeof NVME-OF_POOL_NAME --placment="host01, host02"
+
+Configuration
+=============
+
+Download the ``nvmeof-cli`` container before first use.
+To download it use the following command:
+
+.. prompt:: bash #
+   
+   podman pull quay.io/ceph/nvmeof-cli-rhel9:latest
+
+#. Create an NVMe subsystem:
+
+.. prompt:: bash #
+   
+   podman run -it <qauy.io/ceph/nvmeof-cli-rhel9:latest> --server-address GATEWAY_IP --server-port GATEWAY_PORT 5500 create_subsystem --subnqn SUSYSTEM_NQN --max-namespaces 256
```

Comment:
```suggestion
   podman run -it quay.io/ceph/nvmeof-cli-rhel9:latest --server-address GATEWAY_IP --server-port GATEWAY_PORT 5500 create_subsystem --subnqn SUSYSTEM_NQN --max-namespaces 256
```

Reply:
s/qauy/quay/ remedied. The CLI question is beyond my art, and remains open.

Reply:
``subsystem add`` has now replaced ``create_subsystem`` in the text.
``--subsystem`` has now replaced ``--subnqn`` in the text.